.babelrc

```diff
@@ -3,7 +3,6 @@
     [
       "@babel/preset-env",
       {
-        "modules": false,
```

Comment:
@robertsLando I had to remove this line to make the (existing and new) tests in `src/modules` work using ES6 `import` instead of `require`.
Do you know, if this may have some unintended consequences?

Reply:
Sincerly I don't know. I have checked it here but not 100% sure https://babeljs.io/docs/en/babel-preset-env#modules

Reply:
Yes, that's what I thought too - removing it defaults to "auto" which seems to be the recommended way in combination with babel.